### ISSUE TITLE
MongoDB `bulk:true` run never complete

### ISSUE THREAD

**Issue by ernoaapa** [compose/transporter#152]
I'm trying to get started with Transporter (which feels like awesome!).
Just found out that if I have `bulk:true` in MongoDB configuration, what I use as source, the run never exists. The reason why I would have it in source config is that I didn't want to duplicate configs if I read and write to same database.

If I duplicate the configs so that I have `source_mongodb` and `target_mongodb` and only define `bulk:true` in `target_mongodb` it copies data and exits nicely with code 0.
### Don't work

```
nodes:
  my_mongo:
    type: mongo
    uri: mongodb://192.168.59.103:27017/my_database
    bulk: true
```

```
Source({name:"my_mongo", namespace: 'my_database.SomeCollection'})
  .save({name:"my_mongo", namespace: 'my_database.SomeCollection'});
```

After copying data, it just hangs and never exit.
### Works

```
nodes:
  source_mongo:
    type: mongo
    uri: mongodb://192.168.59.103:27017/my_database
  target_mongo:
    type: mongo
    uri: mongodb://192.168.59.103:27017/my_database
    bulk: true
```

```
Source({name:"source_mongo", namespace: 'my_database.SomeCollection'})
  .save({name:"target_mongo", namespace: 'my_database.SomeCollection'});
```

Copies and exits with code 0.

Transporter version: 0.1.1
Host: OS X El Capitan 10.11.1

**Comment by jipperinbham** [comment]
hi @ernoaapa, I've been able to duplicate the issue locally and will look at getting a fix in.

**Comment by jipperinbham** [comment]
fixed via #211

@ernoaapa we should have a new beta release of v0.1.2 up soon
